tex/latex/biblatex/lbx/romanian.lbx

```diff
@@ -287,7 +287,7 @@
   byorganizer      = {{organizat de}{org\adddotspace de}},
   byauthor         = {{de}{de}},
   byeditor         = {{editat de}{ed\adddotspace de}},
-  bycompiler       = {{compilat de}{comp\adddotspace de}},
+  bycompiler       = {{alcătuit de}{alcăt\adddotspace de}},
```

Comment:
*alcătuit* for `bycompiler` seems to come from a very different root than *redactor* for `compiler`. Is that the natural way to refer to these roles? (In most other languages `compiler` and `bycompiler` use terms that come from the same root.)

Reply:
We don't have `compiler`. I asked and the work of selecting texts and put it together in an anthology is made by a Redactor.
But the process of selecting texts is written in fromtmatter as: Selected text by, or Anthology "X" , alcătuită (put together, made) by

Reply:
I see. It's fine to stick with *alcătuit* if that is the more common way to say this. I only picked up on the apparent inconsistency, but I know nothing about the Romanian language, so obviously I am in no position to recommend anything here.